base/strings/basic.jl

```diff
@@ -715,5 +715,24 @@ unsafe_convert(::Type{Ptr{Int8}}, s::CodeUnits{UInt8}) = unsafe_convert(Ptr{Int8
 Obtain a vector-like object containing the code units of a string.
 Returns a `CodeUnits` wrapper by default, but `codeunits` may optionally be defined
 for new string types if necessary.
+
+# Examples
+```jldoctest
+julia> codeunits("Julia is fast")
```

Comment:
This becomes a bit long (e.g. in the REPL). Perhaps just Julia and maybe some Unicode character so that the length of the CodeUnits isn't exactly the same as the number of characters.

Reply:
Perhaps
```jl
julia> length("¡Julia!")
7

julia> ncodeunits("¡Julia!")
8
```